src/isar_exr/api/energy_robotics_api.py

```diff
@@ -375,8 +422,13 @@ def wake_up_robot(
             result: Dict[str, Any] = self.client.query(
                 dsl_gql(wake_up_robot_mutation), params
             )
-        except Exception as e:
-            raise RobotException(e)
+        except Exception:
+            message: str = "Could not wake up robot"
+            self.logger.error(message)
+            raise RobotException(
```

Comment:
Why not raise `RobotMissionStatusException` and just pass the error description?